app/templates/learn/examples.hbs

```diff
@@ -1 +1,34 @@
-{{outlet}}
\ No newline at end of file
+<h1 id="examples">Examples</h1>
+
+<p>In this section you will find applications that are maintained by the Ember.js teams with the help of contributors. While software is always a work in progress, the goal is to showcase patterns and solutions applied in real-world applications.</p>
+<p>Whether you're simply interested in checking out how some feature is implemented, or you're looking to contribute, one of these projects might pique your interest!</p>
+
+<div class="showcase">
+  {{#each model as |showcase|}}
+    <div class="showcase--item">
+      <div class="showcase--screenshot">
+        <img
+          class="showcase--image"
+          src={{showcase.image.src}}
```

Comment:
Ok so I saw one more thing when checking this out on the deploy-preview link. The images don't load in the ember-app (but are fine on the fastboot load) https://deploy-preview-122--ember-beta-website.netlify.com/learn/examples/

This means you need to use the `{{asset-map}}` for these dynamic images just like we do here: https://github.com/ember-learn/ember-website/blob/master/app/templates/ember-users.hbs#L24

Something like this: 

```
<img
  class="showcase--image"
  src={{asset-map showcase.image.src}}
>
```